internal/spot/types.go

```diff
@@ -91,7 +91,7 @@ type (
 		UtilizeReservedInstances bool
 		FallbackToOnDemand       bool
 		DrainingTimeout          int
-
+		ClusterOrientation       string
```

Comment:
ClusterOrientation is an object, not a string- https://docs.spot.io/api/#tag/Ocean-AWS/operation/OceanAWSClusterCreate